client/src/app/domain/models/projector/projection-default.ts

```diff
@@ -1,23 +1,27 @@
 import { marker as _ } from '@biesbjerg/ngx-translate-extract-marker';
 
-export enum Projectiondefault {
-    agendaAllItems = `agenda_all_items`,
-    topics = `topics`,
-    listOfSpeakers = `list_of_speakers`,
-    currentListOfSpeakers = `current_list_of_speakers`,
-    motion = `motion`,
-    amendment = `amendment`,
-    motionBlock = `motion_block`,
-    assignment = `assignment`,
-    mediafile = `mediafile`,
-    projectorMessage = `projector_message`,
-    projectorCountdown = `projector_countdowns`,
-    assignmentPoll = `assignment_poll`,
-    motionPoll = `motion_poll`,
-    poll = `poll`
-}
+export const PROJECTIONDEFAULT = {
+    agendaAllItems: `agenda_all_items`,
+    topics: `topics`,
+    listOfSpeakers: `list_of_speakers`,
+    currentListOfSpeakers: `current_list_of_speakers`,
+    motion: `motion`,
+    amendment: `amendment`,
+    motionBlock: `motion_block`,
+    assignment: `assignment`,
+    mediafile: `mediafile`,
+    projectorMessage: `projector_message`,
+    projectorCountdown: `projector_countdowns`,
+    assignmentPoll: `assignment_poll`,
+    motionPoll: `motion_poll`,
+    poll: `poll`
+} as const;
 
-export const ProjectiondefaultVerbose: { [key in keyof typeof Projectiondefault]: string } = {
+export type ProjectiondefaultKey = keyof typeof PROJECTIONDEFAULT;
+
+export type Projectiondefault = typeof PROJECTIONDEFAULT[ProjectiondefaultKey];
```

Comment:
I think this should differentiate by more than the case to `PROJECTIONDEFAULT` as they are both exported.